### ISSUE TITLE
Multiple bungee instances and player check on login

### ISSUE THREAD

**Issue by efekt** [BlackDemonZyT/BotSentry#41]
Players are being kicked out multiple times as long as they keep connecting to other bungee instances after last kickout.
Looks like the connection check on login feature is not being synchronized between multiple instances.

**Comment by efekt** [comment]
yep, this is the one, are there any plans to implement it in the near future?

**Comment by Laurenshup** [comment]
I am moving this to a lower priority, as other issues have come up. I can confirm this will be added before the end of this year. Due to the lack of time, this cannot be fully coded in the upcoming version.
